example/patches/react-native-view-shot+4.0.0-alpha.2.patch

```diff
@@ -35,17 +35,4 @@ index bd55b92..6a20e9d 100644
 +        NSString *base64 = [data base64EncodedStringWithOptions: NSDataBase64EncodingEndLineWithLineFeed];
          NSString *imageFormat = ([format isEqualToString:@"jpg"]) ? @"jpeg" : format;
          res = [NSString stringWithFormat:@"data:image/%@;base64,%@", imageFormat, base64];
-       }
-diff --git a/node_modules/react-native-view-shot/src/specs/NativeRNViewShot.ts b/node_modules/react-native-view-shot/src/specs/NativeRNViewShot.ts
-index a6f4c00..1e9e6ce 100644
---- a/node_modules/react-native-view-shot/src/specs/NativeRNViewShot.ts
-+++ b/node_modules/react-native-view-shot/src/specs/NativeRNViewShot.ts
-@@ -2,7 +2,7 @@ import type { TurboModule } from 'react-native';
- import { TurboModuleRegistry } from 'react-native';
-
- export interface Spec extends TurboModule {
--  releaseCapture: () => string;
-+  releaseCapture: (uri: string) => void;
-   captureRef: (tag: number, options: Object) => Promise<string>
-   captureScreen: (options: Object) => Promise<string>;
- }
```

Comment:
Is this patch no longer needed?

Reply:
Doesn't it fix some issues on the IOS platform?

Reply:
What do you mean? It looks like a codegen fix

Reply:
Okay, maybe I overlooked something. I didn't include that patch.

Reply:
It is 100% needed, it was introduced by me and it was indeed fixing their codegen issue. This function had incorrect signature, so I fixed it with a patch.